### DISCUSSION TITLE
Welcome to imagen-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/imagen-pytorch#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by srelbo** [comment]
Hey, @lucidrains thanks for starting this forum and the simply mind-blowing work you are doing! 

Hello everyone, Saravana here from https://elbo.ai. We are a bootstrapped company trying to make products using the best-in-class generative AI. We would love to support Open Source efforts in any way we can.

**Reply by lucidrains** [reply]
@srelbo Hi Saravana and congrats on the launch

I've been playing around with your trained model and it works quite well!

Do you mind if I link to `art.elbo.ai` from the readme, give your website some visibility?

**Reply by srelbo** [reply]
hey Phil @lucidrains 

Thank you for trying us out! We are still in the training process for our Imagen model, what you see on the site is from a latent diffusion model. But we want to replace it with the Imagen model.

Again, thank you so much for the offer, we would love to get some visibility, but it's not running what you built yet. So I don't feel it's right to ask for it. But we will ping you as soon as we are on Imagen.

Also adding Oguz (@oguzelibol )

**Reply by lucidrains** [reply]
ah ok! my mistake! let me know once you do train Imagen successfully and would be happy to help you advertise :smile:

**Reply by srelbo** [reply]
Thank you @lucidrains !
